videoencoder.cpp

```diff
@@ -347,6 +338,14 @@ main (int argc, char *argv[])
 
 	  rgb_to_yuv (&image, ycbcr);
 
+	std::string new_filename = filename;
+
+	new_filename = new_filename.replace(new_filename.end() - 4, new_filename.end(), ".png");
+
+	imwrite(new_filename, image);
+	unlink(filename);
+	link(new_filename.c_str(), filename); #tried with symlink but somehow the link is broken
```

Comment:
this should be a symlink to avoid writing twice... In a quick test, tried it but the link was broken... will take a look into this later.